### PR TITLE
Fix bultin folder not include bug

### PR DESCRIPTION
**Pull request by Quasimewdo**
For the last Beta release, the folder `builtin` containing a list of existing 
GreenpluPython aggregate functions was not included in the package. 
This patch fixes this bug.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -3,4 +3,5 @@
 setuptools.setup(
     name="greenplum-python",
     install_requires=["psycopg2==2.9.3"],
+    packages=setuptools.find_packages(),
```
[the remark this answers is not in the document]
Comment (Quasimewdo) [reply setup.py:6]: Maybe @yihong0618 knows?

[the remark this answers is not in the document]
Comment (beeender) [reply setup.py:6]: @yihong0618

[the remark this answers is not in the document]
Comment (yihong0618) [reply setup.py:6]: you mean greenplum-python ?

[the remark this answers is not in the document]
Comment (yihong0618) [reply setup.py:6]: for now I think we use tox...so the answer seems no,I will do some search.

**Review by yihong0618 — APPROVED** [review]
LGTM

File: setup.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (yihong0618) [reply setup.py:6]: @Quasimewdo can you try add an empty file `__init__.py` in buildin?

[the remark this answers is not in the document]
Comment (Quasimewdo) [reply setup.py:6]: @yihong0618 Yes, already added

**Pull request merged**